src/sentry/lang/native/plugin.py

```diff
@@ -26,6 +26,8 @@
 
 
 class NativeStacktraceProcessor(StacktraceProcessor):
+    supported_platforms = ('cocoa', 'native')
```

Comment:
We'll leave it this explicit for now and add the new `"native"` type wherever necessary. The way platforms are used now has to be changed anyway, so we can put more thought into this later.